src/config/mongoose_config_parser_toml.erl

```diff
@@ -452,46 +459,83 @@ db_tls(Path, #{<<"driver">> := <<"pgsql">>, <<"tls">> := Opts}) ->
     [{ssl, SSLMode}, {ssl_opts, parse_section([<<"tls">> | Path], Opts1)}];
 db_tls(_, _) -> no_tls.
 
-%% path: outgoing_pools.redis.connection.*
+%% path: outgoing_pools.http.*.connection.*
+-spec http_option(path(), toml_value()) -> [option()].
+http_option([<<"host">>|_], V) -> [{server, b2l(V)}];
+http_option([<<"path_prefix">>|_], V) -> [{path_prefix, b2l(V)}];
+http_option([<<"request_timeout">>|_], V) -> [{request_timeout, V}];
+http_option([<<"http_opts">>|_] = Path, V) ->
+    Opts = parse_section(Path, V),
+     [{http_opts, maps:from_list(Opts)}].
```

Comment:
Minor: no reason to indent this. Are you using any automatic indentation helper in your editor?